src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -164,7 +164,7 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
             id="WI_details_reporter_img" />          
           <div class="pull-left marginR10">                          
               <span id="WI_details_reporter_user"> 
-                {{workItem.relationalData?.creator?.attributes.fullName || 'No creator found'}} 
+                {{workItem.relationalData?.creator?.attributes.fullName}} 
               </span>
```

Comment:
@nimishamukherjee , i want to make this PR on hold #605 , waiting for exact UX suggestion.
Please ignore recent changes.
Thanks